### ISSUE TITLE
can't add token to watchlist

### ISSUE THREAD

**Issue by rian64** [MobulaFi/mobula-feedbacks#318]
![20230905_084334](https://github.com/MobulaFi/mobula-feedbacks/assets/122985679/26f89964-70c8-4782-ad17-5df09b1893da)

**Comment by anouci** [comment]
Hey! Can you please verify that you created a watchlist before adding any assets?

**Comment by rian64** [comment]
right, I want to add it to my watchlist and haven't added any assets yet.

**Comment by NBMSacha** [comment]
Hey Rian, could you please share your wallet for me to be able to reproduce this?
